packages/expo-file-system/src/FileSystem.types.ts

```diff
@@ -0,0 +1,63 @@
+export type DownloadOptions = {
+  md5?: boolean;
+  headers?: { [key: string]: string };
```

Comment:
`name`, as in the name of the HTTP header, is a bit more accurate here:

```suggestion
  headers?: { [name: string]: string };
```